content/integrations/java.md

```diff
@@ -103,7 +103,7 @@ You can explicitly specify supplementary tags. For instance, assuming the follow
 
     mydomain:attr0=val0,attr1=val1
 
-It would create a metric called `mydomain` (or some variation depending on the attribute inside the bean) with tags: `attr0:val0, attr1:val1, domain:mydomain, simple:val0, raw_value:my_chosen_value, multiple:val0-val1`. 
```

Comment:
how I get it sorry... I thought it was a cp glitch on my side. 
@kmshultz is it ok for merging for you ?